src/main/java/com/volmit/iris/engine/framework/Engine.java

```diff
@@ -774,9 +777,54 @@ default PlacedObject getObjectPlacement(int x, int y, int z) {
             }
         }
 
+        KList<String> pieces = new KList<>();
+        KList<String> pools = new KList<>();
+        int r = 2;
```

Comment:
Defines the search radius for the jigsaw structure:
1 -> 3x3 (max structure size: 18)
2 -> 5x5 (50)
3 -> 7x7 (82)
4 -> 9x9 (114)
5 -> 11x11 (146)
...